crates/telio-traversal/src/endpoint_providers/stun.rs

```diff
@@ -270,10 +280,10 @@ struct State<Wg: WireGuard, E: Backoff> {
 impl<Wg: WireGuard, E: Backoff> State<Wg, E> {
     async fn start_stun_session(&mut self) -> Result<(), Error> {
         if self.stun_session.is_none() {
+            self.current_timeout = PinnedSleep::new(self.exponential_backoff.get_backoff(), ());
```

Comment:
This should be left as it was STUN_TIMEOUT, backoff should happen in case of failure